tests/framework/console/FakeController.php

```diff
@@ -84,6 +84,10 @@ public function actionAksi6()
         return $this->testArray;
     }
 
+    public function actionAksi7(self $typedArgument, $simpleArgument) {
```

Comment:
The controller was like that and renaming requires much more analysis. I'd say leave it as is for now.